tests/opera/parser/test_tosca.py

```diff
@@ -71,5 +77,27 @@ def test_loads_template_part(self, tmp_path, yaml_text):
             """
         ))
 
-        doc = tosca.load(root.parent, root.name)
+        doc = tosca.load(tmp_path, name)
         assert doc.topology_template.node_templates["my_node"] is not None
+
+    def test_load_from_csar_subfolder(self, tmp_path, yaml_text):
+        name = pathlib.PurePath("sub/folder/file.yaml")
+        (tmp_path / name).parent.mkdir(parents=True)
+        (tmp_path / name).write_text(yaml_text(
+            """
+            tosca_definitions_version: tosca_simple_yaml_1_3
+            imports:
+              - imp.yaml
```

Comment:
The exact location does not really matter since what we are actually testing is that the proper relative path is propagated when we process imports. The rest of the code already handles this (the code is available [here](https://github.com/xlab-si/xopera-opera/blob/e9021b120df76e783baa3f56c9bedca2ecb97847/src/opera/parser/tosca/v_1_3/service_template.py#L71..L73)), we just forgot to add this handling to the parser's entry point.